backend/config/curation-api.yml

```diff
@@ -419,12 +419,16 @@ paths:
                   credentials:
                     type: object
                     properties:
-                      access_key:
```

Comment:
fixing the response to match reality

Reply:
do you want to add the upload path to the response in this PR?

Reply:
It's already there.

Reply:
Oops 😝